compass_app/app/lib/features/results/presentation/result_card.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:compass_app/common/themes/text_styles.dart';
+import 'package:compass_app/common/widgets/tag_chip.dart';
+import 'package:compass_app/features/results/business/model/destination.dart';
+import 'package:flutter/material.dart';
+
+class ResultCard extends StatelessWidget {
+  const ResultCard({
+    super.key,
+    required this.destination,
+  });
+
+  final Destination destination;
+
+  @override
+  Widget build(BuildContext context) {
+    return ClipRRect(
+      borderRadius: BorderRadius.circular(10.0),
+      // TODO: Improve image loading and caching
```

Comment:
Does `pkg:cached_network_image` deliver everything needed?

Reply:
I think so, except if the team has a different suggestion, I'd go with that as well.